src/pubsub/ua_pubsub_networkmessage.c

```diff
@@ -851,7 +851,23 @@ static UA_StatusCode
 UA_NetworkMessage_decodeBinaryInternal(const UA_ByteString *src, size_t *offset,
                                            UA_NetworkMessage* dst) {
 
-    UA_StatusCode rv = UA_NetworkMessage_decodeHeaders(src, offset, dst);
+    UA_StatusCode rv = UA_STATUSCODE_GOOD;
+
```

Comment:
This is now used exclusively in UA_NetworkMessage_decodeBinary.
The functionality can be folded in there.